app/api/account.py

```diff
@@ -25,3 +25,9 @@ async def get_accounts(token: str = Depends(get_current_user)):
 async def get_account(account_id: int, token: str = Depends(get_current_user)):
     results = await read_account(account_id, token)
     return results
+
+
+@router.patch('/accounts/{account_id}', status_code=status.HTTP_200_OK)
+async def patch_account(account: Account, token: str = Depends(get_current_user)):
+    results = await update_account(account, token)
```

Comment:
```suggestion
    updated_account = await update_account(account, token)
```
P2: 단수형인 result 혹은 updated_account가 더 좋을 것 같습니다.